### DISCUSSION TITLE
Ear Trumpet shows Microsoft Logo. How to remove it?

### DISCUSSION THREAD

**Discussion by lev0x1** [File-New-Project/EarTrumpet#1577] [Q&A] [answered]
Why does it show this icon and can i remove it ?

![Screenshot 2024-07-11 233659](https://github.com/user-attachments/assets/98653bb0-a716-4bb9-8309-462ac1c5f925)

**Comment by riverar** [comment]
@lev0x1 What does it say when you move the cursor over it? (Or right click it)

**Reply by lev0x1** [reply]
Its says "Micorosoft-Konto"  (Microsoft-Account in english)
![Screenshot 2024-07-11 235910](https://github.com/user-attachments/assets/baf11c8b-c6e5-4f5d-9359-138875c7d0a7)

**Reply by riverar** [reply]
Can you collect troubleshooting data and look for "Microsoft-Konto" in the list? It should tell you which app/exe is responsible.

![image](https://github.com/user-attachments/assets/14201757-fead-44c2-95ae-af65d479c1c8)

**Reply by lev0x1** [reply]
I copied it from the list:


  Microsoft-Konto
    (Modern) (Inactive) 100% Id: {0.0.0.00000000}.{854afdb4-9d48-4c01-9cd5-ae5b8dcfea92}|#%b{FF24B268-10C4-48BA-B4C2-BDB5B96E6297}|1%b#
    AppId: C:\Windows\SystemApps\Microsoft.Windows.CloudExperienceHost_cw5n1h2txyewy ProcessId: 6532 Alive: True
    IconPath: Microsoft.Windows.CloudExperienceHost_cw5n1h2txyewy!App
    GroupingParam: f7f7ae43-f3fb-4d30-b437-41f354136bc2


It only appeared after I logged in to my microsoft account. Before that I always used an offline account but switching back to the offline account didnt help.

**Reply by riverar** [reply, marked as answer]
Yep, this happens from time to time because these web views make sounds (e.g. little clicks when navigating around). They should disappear from EarTrumpet after a small amount of time.

**Reply by lev0x1** [reply]
It has been there for about 2 weeks. Could it be that an application is using webview and that's why it won't go away?

**Reply by lev0x1** [reply]
You were right its gone now. 
Thanks !
